examples/call-runtime/lib.rs

```diff
@@ -0,0 +1,155 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use ink::primitives::AccountId;
+use sp_runtime::MultiAddress;
+
+#[derive(scale::Encode)]
+enum RuntimeCall {
+    #[codec(index = 4)]
```

Comment:
Just a quick question. Where did you find the index of this call?

Reply:
I checked the `substrate-contracts-node` runtime configuration

Reply:
:white_check_mark: